CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to ProtoSchool
+
+### What is ProtoSchool?
+
+ProtoSchool is an educational community dedicated to making it easy to get started with decentralized technologies. It's an initiative of [Protocol Labs](https://protocol.ai/), which sponsors such open source projects as [IPFS](https://ipfs.io/), [IPLD](https://ipld.io/), [libp2p](https://libp2p.io/), and [Multiformats](https://multiformats.io/). However, we depend on volunteers to develop our coding workshops and make our content available at chapters around the world through live events. We always need more volunteers, and there are always things to do!
+
+ProtoSchool has two main initiatives:
+
+- Building and maintaining workshops that teach about the protocols underlying the decentralized web, hosted at [https://proto.school/#/](https://proto.school/#/)
+- Organizing educational events around the world
+
+As a community, we support the following principles:
+
+- Beginners should feel welcome
+- Chapters should experiment with new ways to engage their community, and share approaches with other chapters over time
```

Comment:
Somewhere, maybe not here but somewhere else, we could describe in a little more detail how the chapters are autonomous.

Reply:
Can you say more me about what they're doing autonomously? 

On a related note, we don't have any description now of what a chapter organizer does in general. In particular, it's not clear whether being a chapter leader means you also need to manage a meetup group and run live events. Could you tell me more about what you envision the responsibilities being based on your experience with NodeSchool?

Reply:
Everything from managing their own website, running their own events, experimenting with different formats.